packages/qulacs/tests/qulacs/circuit/test_noise_model_qulacs.py

```diff
@@ -44,8 +44,10 @@ def gate_info(
             g.get_control_index_list(),
         )
 
+    # the `.get_matrix()` method returns NDArray[complex128, _Shape]
+    # (_Shape should be the first argument)
```

Comment:
I think this is clearer:
```suggestion
    # We need to disable type check due to an error in qulacs type annotation
    # https://github.com/qulacs/qulacs/issues/537
```